ingestion/tests/cli_e2e/test_cli_mysql.py

```diff
@@ -117,12 +117,12 @@ def expected_filtered_schema_excludes() -> int:
 
     @staticmethod
     def expected_filtered_table_includes() -> int:
-        return 78
+        return 81
```

Comment:
Addition of new tables from [here](https://github.com/open-metadata/OpenMetadata/pull/16878/files#diff-9101c22af0510779d6a865b7a89f802187432a3b1f87329736f1877c9c25e152R22) is causing the count change